app/models/merchant.rb

```diff
@@ -1,8 +1,9 @@
 class Merchant < ApplicationRecord
   has_many :invoices
   has_many :items
+  has_many :customers, through: :invoices
 
-  default_scope { order(:id) }
+  # default_scope { order(:id) }
```

Comment:
Did you ever uncomment this? Or should we just remove/change it?

Reply:
I think we will need to remove it so it does not interfere with ordering, unless you know that we need it.